middlewares/validateToken.js

```diff
@@ -0,0 +1,35 @@
+import jwt from 'jsonwebtoken';
+import { UserDao } from '../models';
+
+const { JWT_SECRET_KEY } = process.env;
+
+const validateToken = async (req, res, next) => {
+  try {
+    let bearer, token;
+    const bearerHeader = req.headers.authorization;
+
+    if (bearerHeader) {
+      [bearer, token] = req.headers.authorization.split(' ');
+    } else {
+      const err = new Error('TOKEN_REQUIRED');
+      err.statusCode = 401;
+      throw err;
+    }
```

Comment:
```suggestion
const validateToken = async (req, res, next) => {
  try {
    let token;
    const bearerHeader = req.headers.authorization;

    if (bearerHeader) {
      [_, token] = req.headers.authorization.split(' ');
    } else {
      const err = new Error('TOKEN_REQUIRED');
      err.statusCode = 401;
      throw err;
    }
```
bearer 이라는 변수는 사용하지 않으시는 것 같은데, 이렇게 언더바를 사용하면 이름이 없는 변수를 할당할 수 있습니다.
저도 문법이 정확하지는 않은데, 혹시 이 방법을 한 번 도입해보세요 :)

Reply:
네 bearer이라는 변수는 사용하지 않고 있습니다! 수정하겠습니다!